src/udev/udev-event.c

```diff
@@ -817,6 +824,46 @@ static void event_execute_rules_on_remove(
                 (void) udev_node_remove(dev);
 }
 
+static int udev_event_on_move(UdevEvent *event) {
+        const char *sysname, *rename_to;
+        sd_device *dev;
+        int r;
+
+        assert(event);
+        assert(event->dev);
+        assert(event->dev_db_clone);
+
+        dev = event->dev;
+
+        r = device_copy_properties(dev, event->dev_db_clone);
+        if (r < 0)
+                return log_device_debug_errno(dev, r, "Failed to copy properties from cloned device, ignoring: %m");
+
+        /* Drop previously added property */
+        r = sd_device_get_property_value(dev, "ID_NET_RENAMING_TO", &rename_to);
+        if (r < 0 && r != -ENOENT)
+                return log_device_debug_errno(dev, r, "Failed to get property 'ID_NET_RENAMING_TO', ignoring: %m");
+        if (r == -ENOENT)
+                return 0;
```

Comment:
if you check against -ENOENT first, you can remove the r != -ENOENT check in the other if check...